content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -90,6 +90,10 @@ kubectl apply -f ./my1.yaml -f ./my2.yaml      # create from multiple files
 kubectl apply -f ./dir                         # create resource(s) in all manifest files in dir
 kubectl apply -f https://git.io/vPieo          # create resource(s) from url
 kubectl create deployment nginx --image=nginx  # start a single instance of nginx
+kubectl create job hello                       # create a job which prints "hello world"
+--image=busybox -- echo "Hello World"                                    
+kubectl create cronjob hello --image=busybox   # create a cron job which prints "hello world" for every minute
+--schedule="*/1 * * * *" -- echo "Hello World"       
```

Comment:
@sftim does have some valid concerns. So maybe we should use the format here:

https://github.com/kubernetes/website/pull/24701/files#diff-9b5338cfe4b946f1b17ff38106ad8410dc6687e17e8327d3fc4864a5ee8a48c5R71-R73

Reply:
about line 96 and 97, you can either do 

```
kubectl create cronjob hello --image=busybox --schedule="*/1 * * * *" -- echo "Hello World"
```

or 

```
kubectl create cronjob hello --image=busybox \
  --schedule="*/1 * * * *" -- echo "Hello World"
```

The current format is confusing. It looks like two commands.